web/themes/custom/pineapple/sass/base/_buttons.scss

```diff
@@ -7,28 +7,42 @@
   margin-right: 0;
   margin-left: 0;
   display: inline-block;
+  border: thin solid $primary;
 
   &:hover,
   &:focus {
     background-color: $primary;
     color: $white;
   }
+
+  .button-link {
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+  }
 }
 
 .button--primary {
```

Comment:
See button changes, for comment on this: I'd like to suggest that you create a new basic test page and on that page you add the 3 button variants you can have on your site through the call to action paragraph, If i were you I'd also consider editing the the call to action template and temporarily including all the button variants included in this file so that you can ensure that they all work as they should, i think that having a collection of the buttons in one place will greatly help with finding any issues such as the one noted in notion on the button changes task

Reply:
Good suggestion. Thanks.